---
layout: post
title: "A review of three different studies on hidden markov models for epigenetic problems: A computational perspective"
date: 2024-07-01T12:34:56Z
authors: false
description: false
thumbnail: "/assets/images/gen/literature/entry5.webp"
image: "/assets/images/gen/literature/entry5.webp"
---
# Link to article
[Click to Access](https://doi.org/10.5808/GI.2014.12.4.145){: .btn}

# Citation
Lee, K.-E., & Park, H.-S. (2014). A review of three different studies on hidden markov models for epigenetic problems: A computational perspective. Genomics & Informatics, 12(4), 145â€“150. https://doi.org/10.5808/GI.2014.12.4.145

# Abstract
 > Recent technical advances, such as chromatin immunoprecipitation combined with DNA microarrays (ChIp-chip) and chromatin immunoprecipitation-sequencing (ChIP-seq), have generated large quantities of high-throughput data. Considering that epigenomic datasets are arranged over chromosomes, their analysis must account for spatial or temporal characteristics. In that sense, simple clustering or classification methodologies are inadequate for the analysis of multi-track ChIP-chip or ChIP-seq data. Approaches that are based on hidden Markov models (HMMs) can integrate dependencies between directly adjacent measurements in the genome. Here, we review three HMM-based studies that have contributed to epigenetic research, from a computational perspective. We also give a brief tutorial on HMM modelling-targeted at bioinformaticians who are new to the field.